cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	apiv1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	scale_down_status "k8s.io/autoscaler/cluster-autoscaler/core/scaledown/status"
```

Comment:
nit: `scaledownstatus`? I know some imports use underscores, even in this file, but I find it inconsistent with camelCase convention for naming variables.

Reply:
Yeah, I was on the fence about this. I'll change it to scaledownstatus for consistency sake.